app/views/govuk_publishing_components/components/layout_header/_search.html.erb

```diff
@@ -18,5 +18,10 @@
     id: "site-search-text",
     margin_bottom: 0,
     no_border: true,
+    data_attributes: {
+      track_category: "headerClicked",
+      track_action: "searchSubmitted",
+      track_label: "none",
```

Comment:
will this work without a module?

Reply:
Agreed, I think it needs:

```ruby
data_module: "gem-track-click",
```
to make sure that these attributes are used.

Reply:
Wait, scrub that - the `data-module` is added automatically in the search component itself: https://github.com/alphagov/govuk_publishing_components/blob/a341111172fb799fa79a99b6cc65425f26d98b77/app/views/govuk_publishing_components/components/_search.html.erb#L14

Reply:
Default is set to https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_search.html.erb#L14

Reply:
It's making me think though - is it a good idea to have a default module over there?